File: ruby/7_kyu/cw_mumbling.rb
def accum(s)
  str = ""
    i = 0
    while i < s.length
      str += s[i].upcase + (s[i].downcase * i)
      str += "-" unless i == s.length-1
      i += 1
    end
  return str
end

# https://www.codewars.com/kata/5667e8f4e3f572a8f2000039

File: javascript/7_kyu/cw_mumbling.js
function accum(s) {
	let arr = [];
  for (i = 0; i < s.length; i++) {
    arr.push(s[i].toUpperCase() + s[i].toLowerCase().repeat(i));
  }
  return arr.join("-");
}

// https://www.codewars.com/kata/mumbling/javascript